source/examples/rapids-azureml-hpo/notebook.ipynb

```diff
@@ -204,7 +204,7 @@
     "    # let's see if the compute target already exists\n",
     "    gpu_target = ml_client.compute.get(gpu_compute_target)\n",
     "    print(f\"found compute target. Will use {gpu_compute_target}\")\n",
-    "except:\n",
+    "except MlException:\n",
```

Comment:
The `except:` block of this ends up doing something potentially expensive:

```python
try:
    # let's see if the compute target already exists
    gpu_target = ml_client.compute.get(gpu_compute_target)
    print(f"found compute target. Will use {gpu_compute_target}")
except:
    print("Creating a new gpu compute target...")
    gpu_target = AmlCompute(
        name="rapids-cluster",
        type="amlcompute",
        size="STANDARD_NC12S_V3",
        max_instances=5,
        idle_time_before_scale_down=300,
    )
```

Where `ml_client` is an `azure.ai.ml.MLClient`.

So I think it's worth trying to catch a more specific exception here. That'd reduce the risk of errors like a `NameError` from a type in the preceding code leading to a cluster of machines being spun up unnecessarily.

I chose this particular exception because according to the docs for the `azure-ai-ml` library ([link](https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml.exceptions.mlexception?view=azure-python)), it's *"The base class for all exceptions raised in AzureML SDK code base."*.

I did not actually try running this on Azure, didn't think it warranted that effort. But I'd be happy to go do that if you want.

I did test that this import statement actually works.

```shell
pip install azure-ai-ml
python -c "from azure.ai.ml.exceptions import MlException"
```